monorepo/packages/airdrop/README.md

```diff
@@ -0,0 +1,85 @@
+# Honeycomb Airdrop
+Components to interact with Airdrop Contracts.
+
+## Installation
+`yarn add @pangolindex/airdrop @pangolindex/honeycomb-provider`
+
+or
+
+`npm install @pangolindex/airdrop @pangolindex/honeycomb-provider`
+
+### Install below dependancies as its peer dependancies
+
+```
+react
+react-dom
+@pangolindex/sdk
+```
+
+## Getting Start
+In your main file wrap your app with `HoneycombProvider` and `Web3ReactProvider`:
+
+_Use version **6.0.9** of `@web3-react/core` package._
+
+```tsx
+import { HoneycombProvider } from '@pangolindex/honeycomb-provider';
+import { NetworkContextName, useActiveWeb3React } from '@pangolindex/shared';
+import { Web3ReactProvider, createWeb3ReactRoot } from '@web3-react/core';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+
+const Web3ProviderNetwork = createWeb3ReactRoot(NetworkContextName);
+
+function getLibrary(provider: any): Web3Provider {
+  try {
+    const library = new Web3Provider(provider, 'any');
+    library.pollingInterval = 15000;
+    return library;
+  } catch (error) {
+    return provider;
+  }
+}
+
+// library -> web3.js provider
+// chainId -> chain id with which user is connected
+// account -> user's connected wallet address
+// theme -> optional ( refer Theme guide to customize it )
+ReactDOM.render(
+  <React.StrictMode>
+    <Web3ReactProvider getLibrary={getLibrary}>
+      <Web3ProviderNetwork getLibrary={getLibrary}>
+        <HoneycombProvider library={library} chainId={chainId} account={account} theme={theme}>
+          <App />
+        </HoneycombProvider>
+      </Web3ProviderNetwork>
+    </Web3ReactProvider>
+  </React.StrictMode>,
+  document.getElementById('root'),
+);
+```
+
+## Story Book
+
+1. do `yarn storybook` to start the storybook
```

Comment:
Does this mean that for each package we have to run Storybook? That means we won't be able to run a single Storybook and have all packages run in Storybook at the same time?

Currently all Components are deployed at components.pangolin.exchange. Is this no longer possible?

Reply:
Now yes, I think in future is possible, @SarjuHansaliya can give your more details

Reply:
yes storybook will be separate for now but I think there is composition feature in storybook where we can merge all stories but that we will handle once all essential things are done.